screens/HotspotFormScreen.tsx

```diff
@@ -427,6 +428,19 @@ export const HotspotFormScreen = ({
     }
   };
 
+  const hotspotDetailsValues = [
+    hotspotDetails.address,
+    hotspotDetails.city,
+    hotspotDetails.zip,
+    hotspotDetails.latitude.toString(),
+  ];
+  const address = hotspotDetailsValues
+    .filter(x => !!x)
+    .map(x => x.trim())
+    .join(', ')
+    .trim()
+    .replace(/^, /g, '');
```

Comment:
This code is duplicated from `HotspotDetailScreen.tsx`. Maybe create a helper function?

```typescript
export const formatAddress = (values: Array<string>) => { ... }
```